nvflare/app_opt/he/he_decomposers.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Should remove the he_ prefix in the filename to follow the scheme of the other files in this folder.

Reply:
Removed the prefix.